synth.py

```diff
@@ -0,0 +1,12 @@
+import synthtool as s
+from synthtool import gcp
+
+common = gcp.CommonTemplates()
+
+# ----------------------------------------------------------------------------
+# Add templated files
+# ----------------------------------------------------------------------------
+templated_files = common.py_library()
+s.move(templated_files)
+
+s.shell.run(["nox", "-s", "blacken"], hide_output=False)
```

Comment:
I don't think we need synth here at all:  the templates make too many assumptions which are incorrect for this library.